modules/security/pages/password-policy.adoc

```diff
@@ -0,0 +1,30 @@
+= Password Policy
+:description: Overview of password policy.
+
+You can enable a password policy to mandate the minimal required complexity of all xref:user-access:managing-credentials.adoc#_passwords[user passwords] or xref:user-access:access-control-model.adoc#_acl_password[ACL passwords].
+
+== Password policy requirements
+TigerGraph offers the following password policy.
+When enabled, all passwords must meet the following requirements:
+
+* Have a length between 8 and 128 characters
+* Have at least 1 upper-case letter and 1 lower-case letter
+* Have at least 1 number
+* Have at least 1 special character: `!`, `"`, `#`, `$`, `%`, `&`, `'`, `(`, `)`, `*`, `+`, `,`, `-`, `.`, `/`, `:`, `;`, `<`, `=`, `>`, `?`, `@`, `[`, `]`, `^`, `_`, ```, `{`, `|`, `}`, `~`
+
+When creating or altering a new password or ACL password, you cannot use passwords that do not satisfy the policy if the password policy is enabled.
+
+
+== Enable password policy
+
+To enable the password policy, run the following commands:
+
+[.wrap,console]
+----
+$ gadmin config set Security.UserPasswordPolicy.Enable true <1>
+$ gadmin config set Security.ACLPasswordPolicy.Enable true  <2>
+$ gadmin config apply
+$ gadmin restart gsql
+----
+<1> Setting `Security.UserPasswordPolicy.Enable` to true applies the password policy to passwords for TigerGraph database users.
```

Comment:
also add them to https://docs.tigergraph.com/tigergraph-server/current/reference/configuration-parameters

And mention that the policy won't disable existing user with simple password.

Reply:
yes, they'll be added as I run a script to produce all the new config parameters for every version

Reply:
Still need to mention that
* the policy won't disable existing user with simple password.
* mandatory password policy than always exist: password can't be empty string ("") or "null".

Reply:
Added to the draft